src/AnnotationThread.js

```diff
@@ -438,6 +438,7 @@ class AnnotationThread extends EventEmitter {
      */
     cancelUnsavedAnnotation() {
         if (!util.isPending(this.state)) {
+            this.hideDialog();
```

Comment:
Is it ok to always use hideDialog vs. checking to see if we should use hideMobileDialog?

Reply:
Yep. This method ends up calling hideMobileDialog if the necessary conditions are met and if not, the method short circuits anyways.

Reply:
It ends up catching a possibility for when the normal dialog may not have hidden when it should have